stable/etcd-operator/values.yaml

```diff
@@ -0,0 +1,31 @@
+# Default values for etcd-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: quay.io/coreos/etcd-operator
+  tag: latest
+  pullPolicy: IfNotPresent
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+# etcd-cluster specific values
+cluster:
+  enabled: false
+  name: etcd-cluster
+  size: 3
+  version: v3.1.0-rc.0
+  backup:
+    enabled: true
+    provisioner: kubernetes.io/gce-pd
```

Comment:
This also supports `kubernetes.io/aws-ebs` as mentioned in the README. I'll call that out as a comment here. Let's disable by default